### ISSUE TITLE
[webworker.d.ts] createImageBitmap is missing types for options

### ISSUE THREAD

**Issue by 0xAsimetriq** [microsoft/TypeScript#35545]
**TypeScript Version:**  3.7.3

**Search Terms:** createImageBitmap

**Code** 

```ts
const resized = createImageBitmap(blob, 0, 0, width, height, {
  resizeWidth: width * screenRatio,
  resizeHeight: height * screenRatio
});
```

**Expected behavior:**
Judging by https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap `createImageBitmap` should allow for options object.

**Actual behavior:**
Type definitions in lib.webworker.d.ts currently don't account for this
```ts
declare function createImageBitmap(image: ImageBitmapSource): Promise<ImageBitmap>;
declare function createImageBitmap(image: ImageBitmapSource, sx: number, sy: number, sw: number, sh: number): Promise<ImageBitmap>;
```

**Comment by ydz-one** [comment]
Can I work on this? If so, I have the following questions:

1. Type defs for `createImageBitmap` also exist in `lib.dom.d.ts`- they should be fixed too, right?
2. Should the fix be applied to the LKG in `lib/` or to the generated lib files in the `TSJS-lib-generator` repo, or both?

**Comment by DrXoo** [comment]
Have anyone made this? It seems there is no advance with this issue

**Comment by s4m0r4m4** [comment]
See https://github.com/Microsoft/TypeScript/issues/27475
This seems to be working for me now, no more errors when I run the following (at least in a web worker):

```
const cImageData: ImageData = ...;
const cOpts: ImageBitmapOptions = { ... };

const cRenderedImage: ImageBitmap = await createImageBitmap(
                cImageData,
                0,
                0,
                10,
                10,
                cOpts);
```

**Comment by jakebailey** [comment]
Just looking at old issues; this was fixed in #37464.
